### PR TITLE
Quiz by category

### PR DESCRIPTION
**Pull request by sri-rishi**
Quiz By Category:

Photography Quizzes 

- Wildlife Photography 
- Fashion Photography
- Journalism Photography

### CODE REVIEW

**Review by Ankur9669 — APPROVED** [review]
Code looks Good to me Nice work!!!

File: components/photography.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            <div class="brand-title">
+                <a href="#">
+                    <div class="title font-weight-3">
+                        nakshatra <span class="sub-title">Quiz</span>
+                    </div>
+                </a>
+            </div>
+            <ul class="flex-row nav-cta-box nav-items-box align-center justify-evenly">
+                <li class="nav-items">
+                    <a href="#">Home</a>
```
Comment (pradipgharge) [inline components/photography.html:27]: link the files in a tag

**Review by pradipgharge — APPROVED** [review]
Good to merge! just link the pages in a tags

File: components/photography.html (hunk 1 above)
Comment (sri-rishi) [reply components/photography.html:27]: I will add later, because of file structure

Comment (pradipgharge) [reply components/photography.html:27]: ok great, good to merge !

**Pull request merged**